modules/rpc/README.md

```diff
@@ -0,0 +1,42 @@
+# RPC Module
```

Comment:
This should probably be godoc-style so it shows up on godoc.org under the package